### ISSUE TITLE
Apache Threads Number

### ISSUE THREAD

**Issue by eduardo5000** [GeniusesOfSymfony/WebSocketBundle#234]
Hello,

we have your bundle in our company running. In a physical machine and in a VPS.

A bit later we had run the websocket bundle the number of apache threads grow up so much. We can have 150 apache threads.

Apache in the VPS crash directly.

Do you know why?

What can we do to get things right?

Thx !

**Comment by eduardo5000** [comment]
We are using zmq. And we have 200 users concurrently. The app is sending notifications continously.

We use your bundle to count the number of connected users, too.

Thx!

**Comment by ProPheT777** [comment]
Hey, what do you mean by "apache threads" ? It's the number of opened connections ? 

If you have 200 users it's normal to like 150 connections opened from apache side. May be you need to tune your server configuration to handle it correctly.

What kind resources is becoming busy when load is coming ? (cpu, ram etc) 
What are the specs of your VPS ? 

I have no experience with apache :( So I can't really help you to figured out if problem come from apache config :/)

**Comment by eduardo5000** [comment]
We're tunning apache mpm parameters. We use prefork:

.- StartServers
.- MinSpareServers
.- MaxSpareServers
.- MaxClients
.- MaxRequestsPerChild

We have to restart apache server because is hanging every day.

Thx !

**Comment by ProPheT777** [comment]
Are you using Apache server as reverse proxy for websocket connection ? If
you are not, this problem is not related by the fact that your are using
websocket app.

And if you are using Apache as reverse proxy, show me the socket
configuration, because I know, Apache require a  specific module in order
proxy the socket connection.

Mpm prefork is related to http request only as regular such as fpm.
Websocket server is not served by mpm or fpm.

**Comment by eduardo5000** [comment]
Yes we're using reverse proxy from apache. Our configuration is:

<VirtualHost *:80>
         <Location "/wsauthoban">
                ProxyPass "ws://domain:8090/"
        </Location>
</VirtualHost>

domain is our internal domain.

Apache2ctl -M shows this:

core_module (static)
 so_module (static)
 watchdog_module (static)
 http_module (static)
 log_config_module (static)
 logio_module (static)
 version_module (static)
 unixd_module (static)
 access_compat_module (shared)
 alias_module (shared)
 auth_basic_module (shared)
 authn_core_module (shared)
 authn_file_module (shared)
 authz_core_module (shared)
 authz_host_module (shared)
 authz_user_module (shared)
 autoindex_module (shared)
 deflate_module (shared)
 dir_module (shared)
 env_module (shared)
 filter_module (shared)
 mime_module (shared)
 mpm_prefork_module (shared)
 negotiation_module (shared)
 php5_module (shared)
 **proxy_module (shared)**
 **proxy_wstunnel_module (shared)**
 setenvif_module (shared)
 socache_shmcb_module (shared)
 ssl_module (shared)
 status_module (shared)

We're using reverse proxy because we  can only leave from intranet by port 80.

Thx !
